simulate-travis.py

```diff
@@ -101,30 +250,37 @@ def _remote_or_local(fn, branch='master', remote=False):
     # first, but when using `conda config --add` they should be added from
     # lowest to highest priority.
     for channel in channels[::-1]:
-        sp.run(['conda', 'config', '--add', 'channels', channel], check=True)
+        sp.check_call([bin_for('conda'), 'config', '--add', 'channels', channel])
     print("\nconda config is now:\n")
-    sp.run(['conda', 'config', '--get'])
-    sys.exit(0)
+    sp.check_call([bin_for('conda'), 'config', '--get'])
+
 
 if args.install_requirements:
-    sp.run(
-        [
-            'conda', 'install', '-y', '--file',
-            'https://raw.githubusercontent.com/bioconda/bioconda-utils/'
-            '{0}/bioconda_utils/bioconda_utils-requirements.txt'.format(env['BIOCONDA_UTILS_TAG'])
-        ], check=True)
+    _install_requirements()
+    sys.exit(0)
 
-    sp.run(
-        [
-            'pip', 'install',
-            'git+https://github.com/bioconda/bioconda-utils.git@{0}'.format(env['BIOCONDA_UTILS_TAG'])
-        ],
-        check=True)
+if args.install_alternative_conda:
+    _install_alternative_conda(args.install_alternative_conda, overwrite=args.overwrite)
+    sys.exit(0)
```

Comment:
We could output here a small message, like conda environment successfully created at ...